test/kb_Metrics_server_test.py

```diff
@@ -82,87 +80,92 @@ def getContext(self):
     @unittest.skip("skipped test_run_get_total_logins")
     def test_run_get_total_logins(self):
         m_params = {
-            'epoch_range':(1506815999000, 1514764799000)#(datetime.datetime(2017, 9, 30), datetime.datetime(2017,12,31)
+            # (datetime.datetime(2017, 9, 30), datetime.datetime(2017,12,31)
+            'epoch_range': (1506815999000, 1514764799000)
             #'epoch_range':(1420083768000, 1505876263000)#(datetime.datetime(2015, 1, 1), datetime.datetime(2017,9,20)
         }
         # Second, call your implementation
         ret = self.getImpl().get_total_logins(self.getContext(), m_params)
         print("get_total_logins returns {} records:\n".format(len(ret[0]['metrics_result'])))
```

Comment:
@qzzhang printing is not testing essentially. You should add legit test cases so that other developers can continue contributing.